CommonLibF4/include/RE/Bethesda/MemoryManager.h

```diff
@@ -289,7 +289,10 @@ namespace RE
 
 	[[nodiscard]] inline void* calloc(std::size_t a_num, std::size_t a_size)
 	{
-		return malloc(a_num * a_size);
+		auto ret = malloc(a_num * a_size);
+		if (ret)
+			std::memset(ret, 0, a_num * a_size);
```

Comment:
```suggestion
		if (ret) {
			std::memset(ret, 0, a_num * a_size);
		}
```
Please always use braces for control structures

Reply:
Done.